person.rb

```diff
@@ -1,8 +1,34 @@
-class Person
+class Nameable
+  def correct_name
+    raise NotImplementedError, "#{self.class} has not implemented method #{__method__}"
+  end
+end
+
+class Decorator < Nameable
```

Comment:
- [ ] Class decorator should also have a `correct_name` method where it returns `@nameable.correct_name`

Reply:
Okay. Thank you.